src/LogIn.js

```diff
@@ -0,0 +1,105 @@
+import axios from 'axios';
+import React, { useState, useCallback, useContext } from 'react';
+import { useHistory } from 'react-router';
+import { useCookies } from 'react-cookie';
+import {
+  AppContext,
+  LOG_IN_FAILURE,
+  LOG_IN_REQUEST,
+  LOG_IN_SUCCESS,
+} from './App.js';
+import styled from 'styled-components';
+
+const LogIn = () => {
+  const { dispatch } = useContext(AppContext);
+  const history = useHistory();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [cookies, setCookie, removeCookie] = useCookies(['Token']);
+
+  const handleToSignUp = () => {
+    history.push('/signup');
+  };
+  const handleEmailChange = useCallback((e) => {
+    setEmail(e.target.value);
+  }, []);
+  const handlePasswordChange = useCallback((e) => {
+    setPassword(e.target.value);
+  }, []);
+  const handleFormSubmit = async (e) => {
+    e.preventDefault();
+    dispatch({ type: LOG_IN_REQUEST });
+    try {
+      const response = await axios.post(
+        'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/signin',
+        {
+          email: email,
+          password: password,
+        }
+      );
+      dispatch({ type: LOG_IN_SUCCESS, email: email });
+      setCookie('Token', response.data, { path: '/' });
+      history.push('/vote');
+    } catch (err) {
+      dispatch({ type: LOG_IN_FAILURE, error: err.response });
+      if (err.response.status === 404) {
+        alert('Email 또는 password가 일치하지 않습니다');
+      } else {
+        alert('빈칸없이 작성하세요');
+      }
+    }
+  };
```

Comment:
```suggestion
      if (err.response.status === 400) {
          alert('빈칸없이 작성하세요');
      } else if (err.response.status === 404) {
        alert('Email 또는 password가 일치하지 않습니다');
      } else {
         alert('에러가 발생했습니다.');
      }
    }
  };
```

404 에러랑 빈칸있을 때만 if-else문에 처리해주셨는데, 서버 상의 에러도 있을 테니까 else에 저게 들어가면 안 될수도 있을거 같아요!  저는 switch문 쓰긴 했는데 이렇게 바꿔주면 어떨까요?